unfetter-discover/unfetter-ui
# Inline edit boundries

fixes unfetter-discover/unfetter#553

requires PR https://github.com/unfetter-discover/unfetter-store/pull/60


# Test
```
cd unfetter-ui
npm run test
npm run lint
npm run build:prod
```

1. visit a threat dashboard
2. click the plus fab
3. verify the stepper; modify the boundaries (malware, intrusions, reports) individually
4. verify the side panel, kill chain and other charts refresh accordingly
